src/elements/content-sidebar/activity-feed/activity-feed/ActivityThreadReplies.js

```diff
@@ -53,22 +56,39 @@ const ActivityThreadReplies = ({
                     <LoadingIndicator />
                 </div>
             )}
-            {replies.map((reply: CommentType) => (
-                <Comment
-                    key={`${reply.type}${reply.id}`}
-                    {...reply}
-                    currentUser={currentUser}
-                    getAvatarUrl={getAvatarUrl}
-                    getMentionWithQuery={getMentionWithQuery}
-                    getUserProfileUrl={getUserProfileUrl}
-                    mentionSelectorContacts={mentionSelectorContacts}
-                    onDelete={onDelete}
-                    onEdit={onEdit}
-                    onSelect={onSelect}
-                    permissions={getReplyPermissions(reply)}
-                    translations={translations}
-                />
-            ))}
+            {replies.map((reply: CommentType) =>
+                hasNewThreadedReplies ? (
+                    <BaseComment
```

Comment:
I've seen some places where we take this conditional rendering and separate it out into  a function within the component (for readability), would that be a good idea in these files? i.e.
`{replies.map((reply: CommentType) => renderComment() }